src/appleseed-max-impl/appleseedinteractive/interactivesession.h

```diff
@@ -64,8 +64,8 @@ class InteractiveSession
     void reininitialize_render();
     void end_render();
 
-    void schedule_camera_update(foundation::auto_release_ptr<renderer::Camera>  camera);
-    void schedule_material_update(const IAppleseedMtlMap& material_map);
+    void schedule_camera_update(foundation::auto_release_ptr<renderer::Camera>);
```

Comment:
It's unfortunate to remove the argument names, they are often extremely useful to have when reading the code.